### ISSUE TITLE
Server error

### ISSUE THREAD

**Issue by tamakoshihiroki** [LostMyCode/javascript-deobfuscator#15]
I put the following code which is the simple hello world script obfuscated by obfuscator.io, and try to decode _0x3c5af2.
Tring to deobfuscate caused Server error.

var _0x2d97=['Hello\x20World!','log'];(function(_0x5939d0,_0x2d97cf){var _0x3b7660=function(_0x128fb4){while(--_0x128fb4){_0x5939d0['push'](_0x5939d0['shift']());}};_0x3b7660(++_0x2d97cf);}(_0x2d97,0x1d5));var _0x3b76=function(_0x5939d0,_0x2d97cf){_0x5939d0=_0x5939d0-0x0;var _0x3b7660=_0x2d97[_0x5939d0];return _0x3b7660;};function hi(){var _0x3c5af2=_0x3b76;console[_0x3c5af2('0x0')](_0x3c5af2('0x1'));}hi();

**Comment by LostMyCode** [comment]
Thank you for a report.
[Obfuscator.io owner found this project](https://github.com/LostMyCode/javascript-deobfuscator/issues/12) and then he patched it probably. 

Im gonna unpatch it. Sorry for the inconvenience :(

**Comment by tamakoshihiroki** [comment]
Sorry I couldn't understand.
I thought this is a bug of https://deo.sigr.io/, but you mentioned that this is a bug of obfuscator.io?

**Comment by sanex3339** [comment]
![image](https://user-images.githubusercontent.com/9210534/93659578-614cb980-fa4f-11ea-9787-19f14dfdbb71.png)

**Comment by LostMyCode** [comment]
@tamakoshihiroki 
No. I meant obfuscator.io owner updated algo against this project
